src/lt/objs/files.cljs

```diff
@@ -231,13 +233,13 @@
       (when cb (cb e)))))
 
 (defn trash! [path]
-  (.moveItemTotrash shell path))
+  (.moveItemTotrash electron-shell path))
 
 (defn delete!
   "Delete file or directory"
   [path]
   (if (dir? path)
-    (.rmdirSyncRecursive wrench path)
+    (.rm shell "-rf" path)
```

Comment:
@sbauer322 Using the force option here is a little concerning. Do you have a specific use case for it? Given that [this option ignores write permissions](https://github.com/shelljs/shelljs/blob/35cef9a4ae2b7321bd8cc341a7528ef015bca304/src/rm.js#L125-L127), any bugs (accidental args) to this fn could lead to harmful deletions for users, especially those running as root